packages/affine/widget-scroll-anchoring/src/scroll-anchoring.ts

```diff
@@ -163,9 +163,13 @@ export class AffineScrollAnchoringWidget extends WidgetComponent {
       return;
     }
 
-    blockComponent.scrollIntoView({
-      behavior: 'instant',
-      block: 'center',
+    // use `requestAnimationFrame` to better scroll to the target
+    // because sometimes it is impossible to scroll to the target
```

Comment:
why is it impossible to scroll to the target, but possible in a raf?

Reply:
I need to do more testing

Reply:
https://github.com/facebook/react/issues/23396